github/pulls_test.go

```diff
@@ -59,6 +60,7 @@ func TestPullRequestsService_Get(t *testing.T) {
 
 	mux.HandleFunc("/repos/o/r/pulls/1", func(w http.ResponseWriter, r *http.Request) {
 		testMethod(t, r, "GET")
+		testHeader(t, r, "Accept", mediaTypeGraphQLNodeIDPreview)
```

Comment:
Maybe also do this for `TestPullRequestsService_Create` and `TestPullRequestsService_Edit` below, unless there's a reason not to.

Reply:
Couldn't think of a reason not to, so I did.